pr-data.csv

```diff
@@ -4488,3 +4491,4 @@ https://github.com/zalando/riptide,8277e11fc069d8e24df0d233ef2577cc75659b75,ript
 https://github.com/zalando/riptide,8277e11fc069d8e24df0d233ef2577cc75659b75,riptide-spring-boot-starter,org.zalando.riptide.spring.ObjectMapperOverrideTest.shouldOverride,ID,MovedOrRenamed,,https://github.com/zalando/riptide/commit/cef45a1e325b5b1fdc4c47def3269ad3578b1fe1
 https://github.com/zalando/riptide,8277e11fc069d8e24df0d233ef2577cc75659b75,riptide-spring-boot-starter,org.zalando.riptide.spring.url.UrlResolutionTest.shouldAppendUrl,UD,,,
 https://github.com/zhangxd1989/spring-boot-cloud,e3966d7cefa4fa429d13bbc8de7f4dafbae0de35,registry,cn.zhangxd.registry.ApplicationTests.catalogLoads,ID,,,
+
```

Comment:
Please don't add empty lines.

Reply:
> Did you run multiple times with NonDex to check if the failures are deterministic so likely depend on shuffling rather than another cause? The entries were out of order (so this build failed https://github.com/TestingResearchIllinois/idoft/runs/8078520949?check_suite_focus=true), but I see some changes were made while I was reviewing.

Yes, I ran the NonDex tool multiple times and consistently got the same error.
I have sorted the file now and the pipeline checks have passed.

Reply:
> Please don't add empty lines.

Made changes as suggested.